content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Server Software
+description: A detailed guide on which server software is right for you
+---
+
+import { Card, Cards } from "fumadocs-ui/components/card";
+
+# Introduction
+
+Everybody has different requirements when setting up their server. This guide will help you decide which server software is right for you.
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
+  because server softwares are usually distributed as a `.jar` file.
+</Callout>
+
+# Backend Servers
+
+Backend servers are the most common type of server software.
+There are an incomprehensible amount of backend servers available, however this section will cover the most recommended, stable and widely-used softwares.
```

Comment:
> recommended, stable and widely-used

recommended, stable, and widely-used

Oxford Comma.

Reply:
Already resolved in one of the commits.